src/keycloak/chart/values.yaml

```diff
@@ -149,16 +152,17 @@ service:
   # Session affinity config
   sessionAffinityConfig: {}
 
+# Connection information for external postgres database
 postgresql:
-  # PostgreSQL User to create
-  username: keycloak
-  # PostgreSQL Password for the new user
-  password: keycloak
-  # PostgreSQL Database to create
-  database: keycloak
-  # PostgreSQL host
-  host: postgresql
-  # PostgreSQL port
+  # The username of the database user
+  username: ""
+  # The password of the database user
+  password: ""
+  # Database name of the default JDBC URL
+  database: ""
+  # Hostname of the default JDBC URL
+  host: ""
+  # Port of the default JDBC URL
```

Comment:
Did these new comments come from somewhere else? The username/password ones make sense, the inclusion of "JDBC URL" in the other 3 might be confusing though to an average end user?

Reply:
Yeah, good callout. I copy/pasted directly from the Keycloak [docs](https://www.keycloak.org/server/all-config). I think simplifying them makes sense.

Reply:
Updated descriptions